test/docker.dart

```diff
@@ -0,0 +1,38 @@
+import 'dart:io';
+
+import 'package:docker_process/containers/postgres.dart';
+import 'package:test/test.dart';
+
+const _kContainerName = 'postgres-dart-test';
+
+Future<void> startPostgresContainer() async {
+  setUpAll(() async {
+    if (Platform.environment.containsKey('GITHUB_ACTION')) {
+      // Postgres already running
+      return;
+    }
+
+    final isRunning = await isPostgresContainerRunning();
+    if (isRunning) {
+      return;
+    }
+
+    await startPostgres(
+      name: _kContainerName,
```

Comment:
This is a nice thing to have here, I haven't really thought that you'd match the CI's config, but it makes sense.